misc/scaling/README.md

```diff
@@ -0,0 +1,54 @@
+# Scaling
+An example of AWS Lambda scaling with TOSCA policies.
+
+## Table of Contents
+  - [Description](#description)
+  - [Running with xOpera](#running-with-xopera)
+
+# Description
+This is just an example without real usage and displays the possible scaling TOSCA template for AWS Lambda with two 
+nodes - one for AWS Lambda and the second for monitoring configuration. There are also two policies for scaling up and 
+down, which include two triggers that are invoked after the validation and deployment with opera notify. In the end the 
+example can also be undeployed. The playbooks include just some debug messages with some variables (inputs) from 
+policies or with the `{{ notification }}` variable that holds the contents from the notification file.
```

Comment:
I'd remove jinja template markers from around the variable.

Reply:
Done.

Reply:
Done done done